File: app/src/main/java/com/example/msdc/api/GenreResult.java
package com.example.msdc.api;

import com.google.gson.annotations.SerializedName;

public class GenreResult {
    @SerializedName("id")
    int id;

    @SerializedName("name")
    String name;

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }
}

File: app/src/main/java/com/example/msdc/api/GenreResult.kt
package com.example.msdc.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class GenreResult (
    @field:SerializedName("id") val id: Int? = 0,
    @field:SerializedName("name") val name: String? = null
) : Parcelable